Source/IntegrationTests/TestFiles/LitTests/LitTest/at-attributes/at-attributes-acceptable-builtin.dfy

```diff
@@ -1,6 +1,62 @@
 // RUN: %resolve "%s"
 // Attributes on top-level declarations
 
+@AutoContracts
+class A {
+  ghost var Repr: set<object>
+
+  var x: int
+
+  var y: int
+
+  predicate Valid() {
+    true
+  }
+}
+
+@NativeIntOrReal
+newtype ubyte = x : int | 0 <= x < 256
+
+@NativeInt
+newtype ubyte2 = x : int | 0 <= x < 256
+
+@NativeNone
+newtype ubyte3 = x : int | 0 <= x < 256
+
+@NativeUInt8
+newtype ubyte4 = x : int | 0 <= x < 256
+
+@NativeInt8
+newtype ubyte5 = x : int | 0 <= x < 100
+
+@NativeUInt16
+newtype ubyte6 = x : int | 0 <= x < 256
+
+@NativeInt16
+newtype ubyte7 = x : int | 0 <= x < 256
+
+@NativeUInt32
+newtype ubyte8 = x : int | 0 <= x < 256
+
+@NativeInt32
+newtype ubyte9 = x : int | 0 <= x < 256
+
+//@NativeInt53
+//newtype ubyte10 = x : int | 0 <= x < 256
+
+@NativeUInt64
+newtype ubyte11 = x : int | 0 <= x < 256
+
+@NativeInt64
+newtype ubyte12 = x : int | 0 <= x < 256
+
+//@NativeUInt128
+//newtype ubyte13 = x : int | 0 <= x < 256
+
+//@NativeInt128
+//newtype ubyte14 = x : int | 0 <= x < 256
```

Comment:
Is it intentional that the upper bound on these types are all 256? And is it intentional that some of these are commented out?

Reply:
That all the upper bound is 256 is intentional yes, it made it easier to copy.
The last two are commented out because they are only supported in Rust. The 53 one is supported only in JavaScript. This is one of the rare times where resolution depends on the target compiler. I did not want to omit them for the record but I cannot reasonably add them.